src/System.Data.SqlClient/src/System/Data/SqlClient/TdsEnums.cs

```diff
@@ -531,6 +531,8 @@ public enum FeatureExtension : uint
 
         public const string DEFAULT_ENGLISH_CODE_PAGE_STRING = "iso_1";
         public const short DEFAULT_ENGLISH_CODE_PAGE_VALUE = 1252;
+        public const short DEFAULT_ENGLISH_LOCALE_ID = 1033;
+        public const string DEFAULT_ENGLISH_LOCALE_NAME = "en-US";
```

Comment:
Can we shift these constants in the non-windows LocaleInterop.cs (or any other related name that you choose) to localize the change to non-Windows platforms?